WordPress/Classes/Models/Notifications/Notification.h

```diff
@@ -77,11 +78,13 @@ typedef NS_ENUM(NSInteger, NoteBlockGroupType)
 @property (nonatomic, assign,  readonly) NSNumber               *metaSiteID;
 @property (nonatomic, assign,  readonly) NSNumber               *metaPostID;
 @property (nonatomic, strong,  readonly) NSNumber               *metaCommentID;
+@property (nonatomic, strong,  readonly) NSNumber               *metaReplyID;
 @property (nonatomic, strong,  readonly) NSURL                  *iconURL;
 @property (nonatomic, strong,  readonly) NSDate                 *timestampAsDate;
 
 @property (nonatomic, assign,  readonly) BOOL                   isMatcher;
 @property (nonatomic, assign,  readonly) BOOL                   isComment;
+@property (nonatomic, assign,  readonly) BOOL                   isReplied;
```

Comment:
I just picked *isReplied* so that it would look consistent with the rest, but you're right, hasReply sounds better.

Updating, thanks!